### PR TITLE
[29642] Rename Boards to Forums

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.com/wp/29642

### CODE REVIEW

**Comment by oliverguenther** [comment]
@ulferts I tested the renaming well enough I believe to have this merged. The only critical part is catching all models / references in the migration, you know any other spots that might need to be migrated ? 

Relevant diff section: https://github.com/opf/openproject/pull/7123/files#diff-2741978e7e7065248d2cc6d72b497204

File: lib/open_project/static/links.rb [hunk 1]
```diff
@@ -67,7 +67,7 @@ def links
               label: 'homescreen.links.shortcuts'
             },
             boards: {
-              href: 'https://community.openproject.com/projects/openproject/boards',
+              href: 'https://community.openproject.com/projects/openproject/forums',
               label: 'homescreen.links.boards'
```
Comment (ulferts) [inline lib/open_project/static/links.rb]: This should be `forums`. And also on L69.

**Review by ulferts — CHANGES_REQUESTED** [review]
Found only one place where 'boards' was still lurking. The changes to the db look solid. 👍

File: lib/open_project/static/links.rb (hunk 1 above)
Comment (oliverguenther) [reply lib/open_project/static/links.rb]: Thanks!

**Pull request merged**